action.yml

```diff
@@ -168,6 +168,18 @@ inputs:
     description: 'Display some statistics at the end regarding the stale workflow (only when the logs are enabled).'
     default: 'true'
     required: false
+  ignore-all-activities-before-stale:
```

Comment:
I'm finding this name a little awkward to read. I wonder if something like `updates-remove-stale-label`/`issue-updates-remove-stale` that defaults to `true`?

Reply:
Reverse thinking, maybe it can work yeah.

Reply:
@luketomlinson I even prefer updates-reset-stale-clock, WDYT?

Reply:
Or updates-reset-stale-counter

Reply:
Let's go for `updates-reset-stale` ;)
Thanks

Reply:
207695b217937e0aca56fb009b46dbdce67cd302